backend/tournesol/serializers/contributor_recommendations.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework.serializers import ModelSerializer
+
+from tournesol.models import ContributorRating, ContributorRatingCriteriaScore
+from tournesol.serializers.poll import RecommendationSerializer
+
+
+class ContributorRecommendationsScore(ModelSerializer):
+    """ Outputs the recommending contributor's criteria scores for the given entity """
+    class Meta:
+        model = ContributorRatingCriteriaScore
+        fields = ["criteria", "score"]
+
+
+class ContributorRecommendationsRating(ModelSerializer):
+    criteria_scores = ContributorRecommendationsScore(many=True)
+
+    class Meta:
+        model = ContributorRating
+        fields = ["is_public", "criteria_scores"]
+
+
+class ContributorRecommendationsSerializer(RecommendationSerializer):
+    """ Format the output of the contributor_recommendations views """
+    contributor_ratings = ContributorRecommendationsRating(many=True,
+                                                           source="contributorvideoratings")
+
+    class Meta(RecommendationSerializer.Meta):
+        fields = list((set(RecommendationSerializer.Meta.fields)
+                      - {"criteria_scores"}) | {"contributor_ratings"})
```

Comment:
Do you think we should keep "criteria_scores" (potentially with another name)? For example, if you think it would be nice to have a direct comparison between the contributor's criteria score and the global criteria scores, without having to call the /entities/{uid} API.